EventsExpress.Db/EventsExpress.Db.csproj

```diff
@@ -3,6 +3,13 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
Remove this section, please.

Reply:
Still here. Remove the `    <ItemGroup><Compile Remove="Migrations\20210205202327_AddNotificationType.cs" />
    <Compile Remove="Migrations\20210205202327_AddNotificationType.Designer.cs" />
    <Compile Remove="Migrations\20210205204712_AddUserNotificationType.cs" />
    <Compile Remove="Migrations\20210205204712_AddUserNotificationType.Designer.cs" />
  </ItemGroup> ` section please.